server/sockets.ts

```diff
@@ -408,137 +526,9 @@ export class ServerStream extends Streams.ObjectReadWriteStream<string> {
 
 	_write(data: string) {
 		// console.log('worker received: ' + data);
-		let socket: import('sockjs').Connection | undefined = undefined;
-		let socketid = '';
-		let room: Map<string, import('sockjs').Connection> | undefined = undefined;
-		let roomid = '' as RoomID;
-		let roomChannel: Map<string, ChannelID> | undefined = undefined;
-		let channelid: ChannelID = 0;
-		let nlLoc = -1;
-		let message = '';
```

Comment:
These are gone forever! Always the most awkward parts of mega-switches. I've experimented with several approaches in the past and they've all sucked.